File: n-mod-m.py
import sys

def get_remainder(n, m):
	while n > m:
		n -= m
	return n

with open(sys.argv[1]) as input_file:
	for line in input_file.readlines():
		n, m = list(map(int, line.strip().split(',')))
		print(get_remainder(n, m))

File: n-mod-m.lua
function get_remainder(n, m):
    remainder = n
    while remainder > m do
        remainder = remainder - m
    end
    return remainder
end

for line in io.lines(arg[1]) do
    n = 
    m = 
    io.write(get_remainder(n, m) .. '\n')
end
